src/workers/airtable-sync/actions/payments/updateReimbursementStatus.js

```diff
@@ -0,0 +1,70 @@
+const slackapi = require("~slack/webApi");
```

Comment:
I feel like not worrying about DRYing up the duplication here w/r/t the other message that gets updated by the bot was a good call.

Reply:
Yea it was something I considered. Maybe in the future once this stabalizes a bit!